### DISCUSSION TITLE
HowTo elect one device as reference and overwrite all localDB, localFiles and remoteDB with its content.

### DISCUSSION THREAD

**Discussion by hilsonp** [vrtmrz/obsidian-livesync#194] [Documentation requests]
If at one moment, you get lost with an out of sync system, you must decide on one you want to keep and overwrite the other ones.

It would be really useful to have a kind of Failsafe procedure #192.
With the assumption that the system is made of 2 windows10 computer and one iOS mobile I would like a step by step procedure to:

- Take LaptopA files as reference to rebuild LaptopA local db and overwrite CouchDB completely.
- Dismiss local files and local database on LaptopB and rebuild both local files and local database from the newly rebuild CouchDB.
- Dismiss local files and local database on MobileA and rebuild both local files and local database from the newly rebuild CouchDB.

The target being getting back on track with all devices again in sync.

**Comment by dlphnkck** [comment]
Hello. How about the following steps?

1. LaptopA: LiveSync plugin->Settings->Remote Database configuration->Rebuild everything
2. LaptopB: Settings->Local Database configuration->Fetch rebuilt DB
3. MobileA: Settings->Local Database configuration->Fetch rebuilt DB

**Reply by hilsonp** [reply]
I tried this but this did not work.

**Reply by hilsonp** [reply]
I rote what went wrong here:
https://github.com/vrtmrz/obsidian-livesync/issues/192#issuecomment-1481826034

**Comment by dlphnkck** [comment]
What went wrong? Are there any error messages or error logs?

1. Show debug info: Obsidian->Open command palette->Show debug info
2. LiveSync plugin Make report: LiveSync plugin->Settings->Hatch->Make report
3. (Someday, you may need it. Write logs into the file: LiveSync plugin->Settings->Hatch->Write logs into the file)

**Comment by hilsonp** [comment]
With the help of @vrtmrz I could get my 3 device in sync again and proposed a procedure:
https://github.com/vrtmrz/obsidian-livesync/issues/192#issuecomment-1481867704
